### PR TITLE
Add Jeroo

### PR DESCRIPTION
**Pull request by AndreyNikitin**
https://codio.myjetbrains.com/youtrack/issue/codio-12431

### CODE REVIEW

File: jeroo/playbook.yaml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    # Jeroo
+    - name: Jeroo checkout
+      become: yes
+      become_user: codio
+      git:
+        repo: 'https://github.com/codio/jeroo.git'
+        dest: /home/codio/jeroo/jeroo
+        update: yes
+        force: yes
+        version: 12431_server_save
```
Comment (destitutus) [inline jeroo/playbook.yaml:68]: will it fail when the branch deleted?

File: jeroo/playbook.yaml [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    # Jeroo server
+    - name: Jeroo server checkout
+      become: yes
+      become_user: codio
+      git:
+        repo: 'https://github.com/codio/jeroo-server.git'
+        dest: /home/codio/jeroo/jeroo-server
+        update: yes
+        force: yes
+        version: 12431_server_save
```
Comment (destitutus) [inline jeroo/playbook.yaml:100]: will it fail when the branch deleted?

File: jeroo/playbook.yaml (hunk 1 above)
Comment (AndreyNikitin) [reply jeroo/playbook.yaml:68]: this will change to master on approve

File: jeroo/playbook.yaml (hunk 2 above)
Comment (AndreyNikitin) [reply jeroo/playbook.yaml:100]: this will change to master on approve

**Review by destitutus — DISMISSED** [review]

**Review by destitutus — APPROVED** [review]

**Pull request merged**